core/pkg/service/flag-evaluation/connect_service.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/open-feature/flagd/core/pkg/service/middleware"
```

Comment:
This will make the Linter fail with https://github.com/open-feature/flagd/pull/550